src/bin/monitor.rs

```diff
@@ -39,7 +39,7 @@ fn check_failing_background_jobs(conn: &PgConnection) -> Result<()> {
     // Max job execution time in minutes
     let max_job_time = dotenv::var("MAX_JOB_TIME")
         .map(|s| s.parse::<i32>().unwrap())
-        .unwrap_or(15);
+        .unwrap_or(60);
```

Comment:
This check does not cover the `update_downloads` job, which is allowed to run for much longer. We should consider reducing this back to 15.